src/app/parcels/batch/BatchParcelsReducer.ts

```diff
@@ -212,6 +211,9 @@ export const batchParcelsReducer = async (
         }
         case "override_column": {
             const { newOverrideRow } = action.overrideColumnPayload;
+            if (!newOverrideRow.data) {
+                return state;
+            }
```

Comment:
Should this be `newOverrideColumn`, or have I misunderstood what's going on here?